cvat-ui/src/actions/tasks-actions.ts

```diff
@@ -93,7 +93,7 @@ export function getTasksAsync(query: TasksQuery): ThunkAction<Promise<void>, {},
         }
 
         const array = Array.from(result);
-        const promises = array.map((task): string => (task as any).frames.preview().catch(''));
+        const promises = array.map((task): string => (task as any).frames.preview().catch(() => ''));
```

Comment:
If we need only to ignore raised errors we can use lambda with empty body like this. Guess, it looks more laconic.
```suggestion
        const promises = array.map((task): string => (task as any).frames.preview().catch(() => {}));
```

Reply:
> If we need only to ignore raised errors we can use lambda with empty body like this. Guess, it looks more laconic.

I agree, but `getTasksSuccess` function expects string[] (it should be base64-encoded images) as the second argument:
```ts
 getTasksSuccess(array: any[], previews: string[], count: number, query: TasksQuery)

Reply:
Sure, agree with you